### DISCUSSION TITLE
想请教RFC5780结果与RFC3489不一致是什么情况导致，以谁为准

### DISCUSSION THREAD

**Discussion by reitowo** [HMBSbige/NatTypeTester#203] [Q&A] [answered]
电脑A测试：
RFC3489  
<img width="375" alt="image" src="https://user-images.githubusercontent.com/29846655/212032439-357b85d1-1c79-41c5-a612-7a6ed2b7b871.png">
RFC5780 
<img width="375" alt="image" src="https://user-images.githubusercontent.com/29846655/212032500-2323d721-2b6f-40cd-b439-8e48acee1919.png">
其PortRestrictedCone与RFC5780的Mapping和Filtering结果一致

电脑B测试：
RFC3489
<img width="375" alt="image" src="https://user-images.githubusercontent.com/29846655/212032797-c7fb64b4-2255-44bf-a1f5-4699411ea31b.png">
RFC5780
<img width="378" alt="image" src="https://user-images.githubusercontent.com/29846655/212032870-7083eb96-dc4c-4485-98da-3b1c2e4af518.png">

其Symmetric与 EndpointIndependent+AddressAndPortDependent不一致

且电脑A与电脑B只能建立一次WebRTC P2P通信，断开后再次联通就失败了，推测此结果也和Symmetric+PortRestrictedCone无法NAT穿透是一致的，首次连接能成功也不知道是为什么

**Comment by HMBSbige** [comment, marked as answer]
两种标准测试方法都不一样，并不能一一对应

两种测试方法的流程图都画在 README 里了，具体什么原因结合具体的网络自行分析
